src/matter/common/DeviceTypes.ts

```diff
@@ -5,6 +5,63 @@
  */
 
 export const DEVICE = {
```

Comment:
It would be good to indicate the reference to the specs next to each device, this will help when there is a need to go back the docs.

For instance for ON_OFF_LIGHT:
/** Lighting device that is capable of being switched on or off, see [Matter Device Library R1.0], section 4.1 */

Can be done in a follow-up PR.

Reply:
Will provide now ...

Reply:
Re-Review please :-)